src/v2/Components/BankDebitForm/BankDebitForm.tsx

```diff
@@ -0,0 +1,60 @@
+import { FC, useState } from "react"
+import { PaymentElement, useStripe, useElements } from "@stripe/react-stripe-js"
+import { Box, Button, Spacer } from "@artsy/palette"
+import { useSystemContext } from "v2/System"
+import { LoadingArea } from "../LoadingArea"
+
+export const BankDebitForm: FC = () => {
+  const stripe = useStripe()
+  const elements = useElements()
+  const { user } = useSystemContext()
+  const [isPaymentElementLoading, setIsPaymentElementLoading] = useState(true)
+
+  const handleSubmit = async event => {
+    event.preventDefault()
+
+    if (!stripe || !elements || !user) {
+      return
+    }
+
+    const { error } = await stripe.confirmSetup({
+      elements,
+      confirmParams: {
+        return_url: "https://staging.artsy.net/",
+        payment_method_data: {
+          billing_details: {
+            name: user.name,
+            email: user.email,
+          },
+        },
+      },
+    })
+
+    if (error) {
+      console.log("error", error)
+    }
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <LoadingArea isLoading={isPaymentElementLoading}>
+        {isPaymentElementLoading && <Box height={300}></Box>}
+        <PaymentElement
+          onReady={() => setIsPaymentElementLoading(false)}
+          options={{
+            fields: {
+              billingDetails: {
+                name: "never",
+                email: "never",
```

Comment:
#non-blocking. I wonder if there is a case where buyers want to use a different billing information than the logged-in user. 🤔

Reply:
I thought of that as well... I can bring it up in the next ACH meeting.